LICENSE-binary

```diff
@@ -269,16 +269,19 @@ org.scala-lang:scala-reflect
 org.slf4j:jcl-over-slf4j
 org.yaml:snakeyaml
 
-COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0
+This project bundles the following dependencies under the CDDL 1.0 license.
+See license/LICENSE-CDDL.txt files for details.
 ------------
 javax.servlet:javax.servlet-api
 
-MIT License
+The MIT License (MIT)
+See license/LICENSE-MIT.txt files for details.
 ------------
 org.slf4j:jul-to-slf4j
 org.slf4j:slf4j-api
 
 BSD 3-clause
+See license/LICENSE-BSD-3.txt files for details.
```

Comment:
We still need to keep the leveldbjni license here, because of the copyright information.

Reply:
Got it, we will add it back